machine-learning/scale-scikit-learn.ipynb

```diff
@@ -59,7 +60,7 @@
    "source": [
     "## Distributed Training\n",
     "\n",
-    "<img src=\"images/scikit-learn-logo-notext.png\"/> <img src=\"images/joblib_logo.svg\" width=\"20%\"/> \n",
+    "<img src=\"../images/scikit-learn-logo-notext.png\"/> <img src=\"../images/joblib_logo.svg\" width=\"20%\"/> \n",
```

Comment:
This is tricky.  Jupyter/Jupyterlab would prefer that we keep this as it was.  We'll run into this in other notebooks as well.

Reply:
Just to be clear, this is in the `machine-learning/` sub-directory, so the path to the images directory is up one. Do you still think jupyter wants it as is? Because it's rendering correctly for me with the `../`

Reply:
Well in that case lets leave it as is :) 

+1 from me to merge (though it think that short term we should just merge without coordination).